service/accounting/impl/getCompanyChartOfAccounts.go

```diff
@@ -0,0 +1,54 @@
+package impl
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/calvinkmts/expert-pancake/engine/errors"
+	"github.com/calvinkmts/expert-pancake/engine/httpHandler"
+	db "github.com/expert-pancake/service/accounting/db/sqlc"
+	"github.com/expert-pancake/service/accounting/model"
+)
+
+func (a accountingService) GetCompanyChartOfAccounts(w http.ResponseWriter, r *http.Request) error {
+
+	var req model.CompanyChartOfAccountsRequest
+	httpHandler.ParseHTTPRequest(r, &req)
+
+	errMapRequest := a.validator.Validate(req)
+	if errMapRequest != nil {
+		return errors.NewClientError().WithDataMap(errMapRequest)
+	}
+
+	result, err := a.dbTrx.GetCompanyChartOfAccounts(context.Background(), db.GetCompanyChartOfAccountsParams{
+		CompanyID:   req.CompanyId,
+		AccountName: wildCardString(req.Keyword),
+	})
+	if err != nil {
+		return errors.NewServerError(model.GetCompanyChartOfAccountsError, err.Error())
+	}
+
+	var chart_of_accounts = make([]model.ChartOfAccount, 0)
```

Comment:
kenapa perlu pake make?

Reply:
Ini menjawab response API biar tetep list in case hasil dari query masih kosong